CHANGELOG/CHANGELOG-0.8.md

```diff
@@ -0,0 +1,19 @@
+# Changelog 0.8
+
+## 0.8.0 - 2023-10-26
+
+### Changed
+
+- Updated the Grafana docker image for future plugins. ([#524](https://github.com/antrea-io/theia/pull/524), [@Dhruv-J])
```

Comment:
Remove "for future plugins". As far as users are concerned, the version of Grafana we install as part of Theia has simply been updated to a more recent version.